### PR TITLE
Pull request for #102

### PR DESCRIPTION
**Pull request by ima007**

### CODE REVIEW

**Comment by ima007** [comment]
Hmmm, I'll actually need to check to see why the warning isn't coming out in my last test case.

File: stylesheets/archetype/ui/_breakpoints.scss [hunk 1]
```diff
@@ -2,18 +2,28 @@
 
 // wraps the content block in a breakpoint media query
 // @mixin     breakpoint
-// @param     $key {String} the breakpoint to use
+// @param     $key {String|List} the breakpoint(s) to use
 // @content
 @mixin breakpoint($key) {
```
Comment (eoneill) [inline stylesheets/archetype/ui/_breakpoints.scss]: thanks for taking this on!

consider doing something like...

```scss
// wraps the content block in a breakpoint media query
// @mixin     breakpoint
// @param     $key... {String} the breakpoint(s) to use
// @content
@mixin breakpoint($key...) {
  $breakpoints: ();
  
  @each $name in $key {
    $breakpoint: get-breakpoint($name);
    
    // if we got a valid breakpoint, keep it...
    @if $breakpoint {
      $breakpoints: append($breakpoints, $breakpoint, comma);
    }
    // othwerwise, throw a warning...
    @else {
      @warn "[archetype:breakpoint] could not find a breakpoint for `#{$name}`";
    }
  }
  
  // output all of our collected breakpoints
  @if length($breakpoints) > 0 {
    @media #{$breakpoints} {
      /* [archetype:breakpoint:begin] --- #{$key} --- */
      @content;
      /* [archetype:breakpoint:end] --- #{$key} --- */
    }
  }
}
```

this includes two optimizations:
1) uses a single parameter `$key...` which can take a variable number of arguments, this simplifies the API to `@include breakpoint(key1, key2, key3)` as opposed to having to pass a list literal
2) we keep track of the resolved breakpoints in a comma delimited list, which Sass' `@media` directive is able to handle natively

**Comment by ima007** [comment]
:+1: Thank you for the info! Much cleaner; I've made the updates. 

I also moved the warning message to the get-breakpoint method, and added a debug message. Let me know what your thoughts are on this. I figured that since enable/disable and register breakpoint had their warnings in scope.rb that it also made sense to have 'get-breakpoint' take on the messages as well.

**Comment by eoneill** [comment]
I like it! Thanks for the changes.

**Pull request merged**